.github/ISSUE_TEMPLATE/0-bug-report.yaml

```diff
@@ -132,46 +98,19 @@ body:
         - I don't know
     validations:
       required: false
+
   - type: input
-    id: user-os
-    attributes:
-      label: User's OS?
-      description: What operating system are you running locally?
-      placeholder: "ex. MacOS Big Sur 11.7, or Ubuntu 20.04..."
-  - type: input
-    id: user-browser
-    attributes:
-      label: User's Browser?
-      description: Is the bug occuring in Grafana's user-interface? If so, what browsers are you seeing the problem on? You may choose more than one.
-      placeholder: "ex. Google Chrome Version 112.0.5615.137 (Official Build) (arm64)..."
-  - type: dropdown
```

Comment:
Do we want to remove this toggle (regression)?

This might be useful to know still in the new template